newIDE/app/src/ProjectsStorage/GoogleDriveStorageProvider/index.js

```diff
@@ -404,48 +401,51 @@ export default ({
                 closeDialog();
                 resolve({ wasSaved: false, fileMetadata });
               }}
-              onSave={({ selectedFileOrFolder, newFileName }) => {
+              onSave={async ({ selectedFileOrFolder, newFileName }) => {
                 const content = serializeToJSON(project);
 
                 if (options && options.onStartSaving) options.onStartSaving();
 
+                const googleUser = await authenticate();
                 if (selectedFileOrFolder.type === 'FOLDER') {
-                  return authenticate().then(googleUser =>
-                    createNewJsonFile(
-                      selectedFileOrFolder.id,
-                      newFileName
-                    ).then(newFileId =>
-                      patchJsonFile(newFileId, googleUser, content).then(() => {
-                        closeDialog();
-                        resolve({
-                          wasSaved: true,
-                          fileMetadata: {
-                            fileIdentifier: newFileId,
-                            lastModifiedDate: Date.now(),
-                          },
-                        });
-                      })
-                    )
+                  const newFileId = await createNewJsonFile(
+                    selectedFileOrFolder.id,
+                    newFileName
                   );
+                  const newFileMetadata = {
+                    fileIdentifier: newFileId,
+                  };
+                  await options.onMoveResources({ newFileMetadata });
```

Comment:
I didn't realize moving project from GoogleDrive to Cloud would be available! So nice

Reply:
This is a actually "easy" in this direction because no custom resources were allowed, just public URLs, so there is nothing to move 😅
The other direction, similarly, will only work if you have public URLs. If you use a resource uploaded to GDevelop cloud, you will get an error for this/these resource(s) (that you can ignore).

In practice, we'll just strongly recommend to use GDevelop cloud. Download a copy will allow you to download the game including the resources you uploaded, so good to do a backup. (to import it back, we either need a specific storage provider to open from a zip file, or we need to ask to go through the desktop app).